pulp_rpm/app/tasks/publishing.py

```diff
@@ -329,6 +335,30 @@ def create_rempomd_xml(packages, publication, extra_repomdrecords, sub_folder=No
             mod_yml.write(default._artifacts.get().file.read())
             has_modules = True
 
+    # Process comps
+    comps = libcomps.Comps()
+    for pkg_grp in PackageGroup.objects.filter(
+            pk__in=publication.repository_version.content):
+        group = pkg_grp.pkg_grp_to_libcomps()
+        comps.groups.append(group)
+        has_comps = True
```

Comment:
@has_comps@ needs to default to False somewhere outside of for loops, because if there is no comps.xml in a repo, @has_comps@ won't be set at all. And the failure in travis is about that, btw.

Reply:
oops!  thanks for spotting this!